<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>653</width>
    <height>538</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>GoldenDict</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QHBoxLayout" name="horizontalLayout_2" >
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2" >
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" >
        <item>
         <widget class="QLabel" name="groupLabel" >
          <property name="text" >
           <string>Look up in:</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="groupList" >
          <property name="sizeAdjustPolicy" >
           <enum>QComboBox::AdjustToContents</enum>
          </property>
          <property name="duplicatesEnabled" >
           <bool>true</bool>
          </property>
          <property name="frame" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLineEdit" name="translateLine" >
        <property name="minimumSize" >
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="baseSize" >
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="palette" >
         <palette>
          <active>
           <colorrole role="Base" >
            <brush brushstyle="SolidPattern" >
             <color alpha="255" >
              <red>254</red>
              <green>253</green>
              <blue>235</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="Base" >
            <brush brushstyle="SolidPattern" >
             <color alpha="255" >
              <red>254</red>
              <green>253</green>
              <blue>235</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="Base" >
            <brush brushstyle="SolidPattern" >
             <color alpha="255" >
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
        <property name="frame" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QStackedWidget" name="stackedWidget" >
        <property name="lineWidth" >
         <number>0</number>
        </property>
        <property name="currentIndex" >
         <number>0</number>
        </property>
        <widget class="QWidget" name="page" >
         <layout class="QVBoxLayout" name="verticalLayout_3" >
          <property name="margin" >
           <number>0</number>
          </property>
          <item>
           <widget class="QListWidget" name="wordList" >
            <property name="palette" >
             <palette>
              <active>
               <colorrole role="Base" >
                <brush brushstyle="SolidPattern" >
                 <color alpha="255" >
                  <red>254</red>
                  <green>253</green>
                  <blue>235</blue>
                 </color>
                </brush>
               </colorrole>
              </active>
              <inactive>
               <colorrole role="Base" >
                <brush brushstyle="SolidPattern" >
                 <color alpha="255" >
                  <red>254</red>
                  <green>253</green>
                  <blue>235</blue>
                 </color>
                </brush>
               </colorrole>
              </inactive>
              <disabled>
               <colorrole role="Base" >
                <brush brushstyle="SolidPattern" >
                 <color alpha="255" >
                  <red>255</red>
                  <green>255</green>
                  <blue>255</blue>
                 </color>
                </brush>
               </colorrole>
              </disabled>
             </palette>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_2" >
         <layout class="QVBoxLayout" name="verticalLayout_4" >
          <property name="margin" >
           <number>0</number>
          </property>
          <item>
           <widget class="QTabWidget" name="tabWidget_2" >
            <property name="tabPosition" >
             <enum>QTabWidget::West</enum>
            </property>
            <widget class="QWidget" name="tab_2" >
             <attribute name="title" >
              <string>Tab 1</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_5" >
              <property name="margin" >
               <number>0</number>
              </property>
              <item>
               <widget class="QListWidget" name="listWidget" />
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="tab_3" >
             <attribute name="title" >
              <string>Tab 2</string>
             </attribute>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget" >
      <property name="currentIndex" >
       <number>0</number>
      </property>
      <property name="iconSize" >
       <size>
        <width>16</width>
        <height>16</height>
       </size>
      </property>
      <property name="elideMode" >
       <enum>Qt::ElideRight</enum>
      </property>
      <widget class="QWidget" name="tab" >
       <attribute name="title" >
        <string>Welcome!</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_4" >
        <property name="margin" >
         <number>0</number>
        </property>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
   <zorder>tabWidget</zorder>
   <zorder>view</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>653</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>&amp;File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_Edit" >
    <property name="title" >
     <string>&amp;Edit</string>
    </property>
    <addaction name="sources" />
    <addaction name="groups" />
   </widget>
   <addaction name="menuFile" />
   <addaction name="menu_Edit" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <action name="action_Preferences" >
   <property name="text" >
    <string>&amp;Preferences...</string>
   </property>
  </action>
  <action name="sources" >
   <property name="text" >
    <string>&amp;Sources...</string>
   </property>
  </action>
  <action name="groups" >
   <property name="text" >
    <string>&amp;Groups...</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>translateLine</tabstop>
  <tabstop>tabWidget</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc" />
 </resources>
 <connections/>
</ui>
